lib/python/pyflyby/_py.py

```diff
@@ -949,10 +949,9 @@ def _handle_user_exception(exc_info=None):
         # *** Run debugger. ***
         debugger(exc_info)
     # TODO: consider using print_verbose_tb(*exc_info)
-    print("Traceback (most recent call last):", file=sys.stderr)
+    print(exc_info)
```

Comment:
Probably need to delete this? `exc_info` is a tuple of three things that don't make sense to print, and `print` goes to stdout, not stderr.